public/table.js

```diff
@@ -13,6 +13,10 @@ function load() {
 }
 function addData() {
     let data = document.getElementById('URL').value.toString().trim();
+    if (data.startsWith("<:")){

```

Comment:
although rare there are emojis that will start with `<a:` instead of `<:` 
for example `<a:neuroSoCute:1111022323219304608>`